scripts/windows/install-wsl.ps1

```diff
@@ -0,0 +1,48 @@
+# //////////////////////////////////////////////////////////////////////
+# install-wsl.ps1
+# See https://docs.microsoft.com/en-us/windows/wsl/install-win10 for the official story behind this code
+
+Param([ValidateSet("EnableWSL-01", "EnableVMPlatform-02", "InstallLinuxUpdatePackage-03")]$Step = "EnableWSL-01")
+
+$workDir = (Join-Path ([System.IO.Path]::GetTempPath()) rdinstall)
+New-Item -ItemType Directory -Force -Path $workDir
+
+$logFile = (Join-Path $workDir restarts.txt)
+$wslMsiFile = (Join-Path $workDir wsl_update_x64.msi)
+
+$script = $myInvocation.MyCommand.Definition
+$scriptPath = Split-Path -parent $script
+. (Join-Path $scriptpath restart-helpers.ps1)
+$sudoInstallScript = (Join-Path $scriptPath sudo-install-wsl.ps1)
+
+#Requires -RunAsAdministrator
+Clear-Any-Restart
+
+if ($Step -eq "EnableWSL-01") {
+  Write-Output "Doing Step EnableWSL-01"
+  Write-Output "Doing Step EnableWSL-01" | Out-File $logFile
+  dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+  Restart-Machine-With-Resume-Command $sudoInstallScript "EnableVMPlatform-02" "installation (step 2)"
```

Comment:
I _think_ `HKLM\…\Run` already runs with administrator privileges? That is, I don't think we need to raise an elevation prompt?

Of course, we can also bullet-proof things by checking in the sudo script to see if we're already elevated and not prompt in that case…

Reply:
That's not my experience, but I'm not 100% sure. The problem being that error messages would flash too quickly to read, even when I used a screen capture program to grab the output -- the resolution was too coarse to see the message.

Reply:
The PowerShell script we use for the installer seems to be okay, so I assumed it was running elevated…

Reply:
Maybe there's a better way of doing this, but if I take the `-Verb Runas` out of `sudo-install.ps1` I get a WindowsTerm window quickly flash with some red text. I was able to see the error message at some point, and it was complaining that the runner didn't have sufficient privileges.

Reply:
Verified that privileges are needed.